rlcard/games/gin_rummy/gin-rummy-guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Gin Rummy Guidelines
```

Comment:
Can we move this document to https://github.com/datamllab/rlcard/blob/master/docs/games.md#gin-rummy